group_vars/maas_region_controller

```diff
@@ -20,6 +23,8 @@ maas_open_tcp_ports:
 - 5432 			# postgres
 - 5443 			# MAAS https port
 - 8000
+- "{{ maas_proxy_port if 'maas_proxy' in group_names }}"
+- "{{ maas_proxy_postgres_port if 'maas_proxy' in group_names }}"
```

Comment:
The firewall task to open ports fails if maas_proxy is not set, I don't think Ansible likes empty items in an itterable. The solution might just be a simple case of filtering the list at the task to only include non-empty values with something like:
`with_items: "{{ maas_open_tcp_ports | select() }}"`